src/ui/ui_settings/__tests__/index.js

```diff
@@ -2,7 +2,7 @@ import { isEqual } from 'lodash';
 import sinon from 'sinon';
 import expect from 'expect.js';
 import init from '..';
```

Comment:
__issue:__ It looks like unit test are failing because of the fact that `init` is not what we would expect now. It should probably be something like this `import { uiSettingsMixin as init } from '..';`